{
	"name": "cpe:/a:ibm:cognos_analytics:11.2.2",
	"title": [
		{
			"text": "IBM Cognos Analytics 11.2.2",
			"lang": "en-US"
		}
	],
	"references": [
		{
			"text": "Advisory",
			"href": "https://exchange.xforce.ibmcloud.com/vulnerabilities/283364"
		}
	],
	"cpe_23_item": {
		"name": "cpe:2.3:a:ibm:cognos_analytics:11.2.2:*:*:*:*:*:*:*"
	}
}
